test/controllers/SignedOutControllerSpec.scala

```diff
@@ -17,23 +17,24 @@
 package controllers
 
 import base.SpecBase
-import play.api.http.Status
 import play.api.test.FakeRequest
-import play.api.test.Helpers.{GET, defaultAwaitTimeout, route, status, writeableOf_AnyContentAsEmpty}
+import play.api.test.Helpers._
 
 class SignedOutControllerSpec extends SpecBase {
 
   "SignedOut Controller" must {
 
-    "return OK and the correct view for a GET" in {
+    "redirect to feedback survey" in {
```

Comment:
Nit: Technically, this service only redirects to `bas-gateway-frontend`, which in turn redirects to feedback survey.